src/App.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import logo from "./logo.svg";
+import "./App.css";
+import gameMapCreator from "./game-logic/gameMap";
+import lifeRules from "./game-logic/gameLoop";
+
+function App() {
```

Comment:
Oh so is this what came with the React App template?  I think this will be rendered to the <root>? if so just a reminder to update before submission!